core-java/src/main/java/com/baeldung/nulls/APIContracts.java

```diff
@@ -0,0 +1,29 @@
+package com.baeldung.nulls;
+
+public class APIContracts {
+
+    /**
+     * Prints the value of {@code param} if not null. Prints {@code null} otherwise.
+     * @param param
+     */
+    public void print(Object param) {
+        System.out.println("Printing " + param);
+    }
+
+    /**
+     *
+     * @return non null result
+     * @throws Exception - if result is null
+     */
+    public Object process() throws Exception {
+        Object result = doSomething();
+        if (result == null)
+            throw new Exception("Processing fail. Got a null response");
```

Comment:
It's a matter of style, but it's generally considered good practice to use curly braces with if/then/else, even when only one statement is involved.